stack/rush-stack-compiler-shared/src/shared/ToolPackages.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+const { ToolPaths } = require('./ToolPaths');
+
+let _typescript;
+Object.defineProperty(
+  module.exports,
+  'Typescript',
+  {
+    configurable: false,
+    get: () => {
+      if (!_typescript) {
+        _typescript = require(ToolPaths.typescriptPackagePath);
```

Comment:
This seems hacky. Have you looked at [import-lazy](https://www.npmjs.com/package/import-lazy)?  I suspect that sprinkling a few import-lazy wrappers throughout our projects could greatly increase our startup time.  For example, `rush -h` does not need to eval every last CommonJS dependency of Rush.

Reply:
Updated to use `import-lazy`